keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -884,19 +891,52 @@ However, as disabling taint-manager is deprecated it is not a concern for this
 KEP.
 
 For Alpha, we implemented a fix for this issue by setting the pod phase as `Failed`
```

Comment:
I think it's better to put this in two bullet points, one describing the fix for scheduled pods when kubelet disappears and one for terminating unscheduled pods.

Reply:
I see, I restructured the content, but have three points. Hope it makes sense. Also, dropped some less relevant parts for brevity, will re add it requested.